quobyte_driver.go

```diff
@@ -44,28 +44,20 @@ func (driver quobyteDriver) Create(request volume.Request) volume.Response {
 	defer driver.m.Unlock()
 
 	user, group := "root", "root"
-	configurationName := "BASE"
+	configurationName := driver.configName
 	retryPolicy := "INTERACTIVE"
-	tenantID := "default"
+	tenantID := driver.tenantID
```

Comment:
Point to the `default` tenant if the opt is not set?

Reply:
Sadly does not work here. We need the default tenant id which, differing from the default tenant name, is generated randomly with each installation. There's no way to provide a working default here.